browser/js/product-details/product-details.js

```diff
@@ -108,9 +105,29 @@ app.config( function ( $stateProvider ) {
       $scope.newReviewForm.$setUntouched();
       $scope.hideForm();
     });
+  }
+
+  $scope.getId = function (item){
+    return item.user;
+  }
 
+  $scope.getEmail = function(user){
+    return UserFactory.getEmail(user);
   }
```

Comment:
Helper functions shouldn't be placed on the scope unless they need to be visible to the user.